lxd/db/cluster/config.mapper.go

```diff
@@ -39,12 +39,36 @@ func GetConfig(ctx context.Context, tx *sql.Tx, parent string) (map[int]map[stri
 		fillParent[i] = strings.Replace(parent, "_", "s_", -1) + "s"
 	}
 
-	sqlStmt, err := prepare(tx, fmt.Sprintf(configObjectsLocal, fillParent...))
-	if err != nil {
-		return nil, err
+	queryStr := fmt.Sprintf(configObjectsLocal, fillParent...)
+	queryParts := strings.SplitN(queryStr, "ORDER BY", 2)
+	args := make([]any, 0, DqliteMaxParams)
```

Comment:
We should not pre-allocate a slice of length `DqliteMaxParams` as this is already quite large (255) and will increase significantly in the future. I don't see why this is needed TBH. As `go-dqlite` will now fail if we pass in an args that is longer than it can handled.

Reply:
Feels like this should be set to the capacity of `len(filters)` instead.